### ISSUE TITLE
Thank you very much a lot for your work!

### ISSUE THREAD

**Issue by yekeping** [davidsandberg/facenet#215]
First,thanks to the extreme for your work!

but,I start a new traing schedule using all your framework except 
    1.the file path contaning the picture
    2.without cleaning the CASIA image because I don't know how to do it
    3.I replace the LFW picture path and pairs.txt with mine,because I want to test the accuacy of my pictures.
(should I modify some other parameters?)

I get an very awesome result,could you please give me some suggestions?

![Uploading 2.jpg…]()

![Uploading 1.jpg…]()
(the title LFW is wrong,it's my picture performace)

**Comment by yekeping** [comment]
![2](https://cloud.githubusercontent.com/assets/22095268/24179356/a3451f6c-0ee9-11e7-85fb-de35b72b6107.jpg)
![1](https://cloud.githubusercontent.com/assets/22095268/24179357/a3e4d00c-0ee9-11e7-9c28-991a64c179a0.jpg)

**Comment by yekeping** [comment]
could anyone help me?(the tiltle LFW is wrong,I have replaced Lfw picture with mine)

**Comment by ugtony** [comment]
Hi, @yekeping 
I don't see any problem. What are you trying to ask specifically?
If you replace picture_path and pairs.txt correctly, it should work.

**Comment by yekeping** [comment]
@ugtony ，the accuracy is only 80% on my dataset after 80 epoches' training,I wonder if there are any way to improve my perfomance and how.should I modify the parameter.

thank you very much!

**Comment by ugtony** [comment]
How big is your training dataset? <= may be too small
Does your validation dataset have similar attributes as the training dataset does? <= if not, sampled from surveillance videos for example, 80% is not surprising.
It would be helpful if you compare the training accuracy and validation accuracy.

**Comment by yekeping** [comment]
@ugtony my training dataset is CASIA(10514 classes and more than 450000 pictures)
                and the validation dataset  is alighed with the method provided by the author,as  david says in validation on lfw,the validation dataset should be alighed in the same way as  the training dataset(mine is CASIA).
                 I see david get 90% accuracy on LFW in less than 10 epoches,my dataset is lower than 85% even after 80 epoches.I have no idea.
                 thanks for your reply!

**Comment by davidsandberg** [comment]
To be able to troubleshoot this I think you should split the problem in two, i.e if your model works on the normal LFW dataset or the if there is a problem with your test set. 
Did you align your test dataset using MTCNN as well? If not that is probably the problem.

**Comment by yekeping** [comment]
@davidsandberg 
Yes,I align my dataset in the same way as the traing set(use your code^_^,the reason why I use the CASIA for training instead of mine is that I only have two pictures in one class.I guess if I find a way to train the model with my dataset(190000 classes,every class has only two pictures),the accuracy will improve,but I have no idea to train such a model with my dataset,can you give me a  suggestion?)

I havn't used my trained model on normal Lfw because my computer is still running in differene learning rate!

and can you tell me how to train a model when the dataset has a lot of classed,but every classes has only two pictures!

thanks for 1000 times!

**Comment by yekeping** [comment]
@davidsandberg
and if use your  trained model which gets 99% on LFW,the accuracy on my dataset is 79%

**Comment by davidsandberg** [comment]
Hi @yekeping,
To me it sounds like your test dataset for some reason is more difficult than LFW. It could also be that it contains label noise.

**Comment by Victoria2333** [comment]
@yekeping, I see your dataset is small sample dataset with each class of only two pictures , I wonder how you solve this problem. I've been solving small sample problems. Waiting for your reply...
